src/Augurk.Specifications/Gherkin/SequenceDiagrams.feature

```diff
@@ -0,0 +1,31 @@
+﻿@augurk @documentation @childOf:UML
+Feature: Sequence Diagrams
+
+Within feature descriptions one can use sequence diagrams to provide additional context for a specific feature. This works by parsing *sequence diagram
+code blocks* with an embedded [js-sequence-diagrams](https://bramp.github.io/js-sequence-diagrams/) parser. For example, the following markdown will lead
+to the graph below it (note that the language name *sequence* is case-sensitive).
+
+    ```sequence
+	Label: A simple Sequence Diagram
+    Foo->Bar: Hello
+    ```
+```sequence
+Label: A simple Sequence Diagram
+Foo->Bar: Hello
+```
+
+Using the syntax of js-sequence-diagrams we can also create more complex diagrams, such as this one:
+
+```sequence
```

Comment:
I wonder whether this is the right example. There is little value to showing the different line types when you cannot see the notation. Shouldn't we take the same approach as with the UML feature and show an actual sequence diagram?